sdk/servicebus/service-bus/src/util/sasServiceClientCredentials.ts

```diff
@@ -57,4 +58,8 @@ export class SasServiceClientCredentials implements ServiceClientCredentials {
     webResource.withCredentials = true;
     return webResource;
   }
+
+  getToken(audience: string): AccessToken {
+    return new SharedKeyCredential(this.keyName, this.keyValue).getToken(audience);
```

Comment:
We don't need to create a new SharedKeyCredential everytime `getToken()` is called. Consider creating once in the constructor and storing as private property

Reply:
Hmm, thought about it.

Reply:
Done!